torch/csrc/jit/frontend/sugared_value.cpp

```diff
@@ -271,6 +282,17 @@ void SimpleValue::setAttr(
             << "Initialize the field at the top level first";
       }
     } else {
+      // Check and see if it's a setter attribute.
+      std::string setter_name = field + "_setter";
+      auto* setter = classType->findMethod(setter_name);
+      if (setter) {
+        auto& g = *m.graph();
```

Comment:
Same here

Reply:
Oops, I meant to comment. So what does it mean to use `MethodValue` here? I thought that the point of this function is to desugar the `setAttr` call correctly and convert it to the appropriate sequence of nodes in the graph?